cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -1336,15 +1355,15 @@
         },
         "TemplateURL": {
           "Fn::Sub": [
-            "https://${s3_domain}/${AWS::Region}-aws-parallelcluster/templates/efs-substack-${version}.cfn.json",
+            "https://s3.${AWS::Region}.${s3_url}/${AWS::Region}-aws-parallelcluster/templates/efs-substack-${version}.cfn.json",
             {
-              "s3_domain": {
-                "Fn::If": [
-                  "GovCloudRegion",
+              "s3_url": {
```

Comment:
If possible let's define s3_url suffix or better the whole s3_url at the beginning once and then use it with Ref without to repeat the FindInMap construct multiple times.

Reply:
If I do, it needs to be a parameter 😌